client/src/components/Reviews.jsx

```diff
@@ -1,5 +1,82 @@
 import React from 'react';
 import StarRatingComponent from 'react-star-rating-component';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  font-size: 14px;
```

Comment:
Another "not worth it for now but for future review": `em` and `rem` are considered best practice for CSS these days and help you make better scaled designs: https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97